core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/stack/cluster/gateway/GatewayJsonValidator.java

```diff
@@ -20,32 +19,44 @@
 @Component
 public class GatewayJsonValidator implements Validator<GatewayJson> {
 
-    @Inject
-    private GatewayConvertUtil gatewayConvertUtil;
+    private final GatewayConvertUtil gatewayConvertUtil;
+
+    public GatewayJsonValidator(GatewayConvertUtil gatewayConvertUtil) {
```

Comment:
Inject by constructor is not common in project.

Reply:
I had to do it because there is no other way to properly test ```ClusterRequestToGatewayConverter```

Reply:
solved, thx to SpringJUnit4ClassRunner